presto-docs/src/main/sphinx/security/kerberos-configuration.fragment

```diff
@@ -0,0 +1,24 @@
+Kerberos Configuration
+^^^^^^^^^^^^^^^^^^^^^^
+
+Kerberos needs to be configured on the |subject_node| At a minimum, there needs
+to be a ``kdc`` entry in the ``[realms]`` section of the ``/etc/krb5.conf``
```

Comment:
kerberos-services.fragment defines KDC at its first use with a :abbr: It's the first thing that we include in both cli.rst and server.rst, so we should be good? I'll ping you if that changes for some reason when I make the upstream pull request and we'll figure out a solution.